package.json

```diff
@@ -87,8 +87,8 @@
     "try-require": "^1.2.1",
     "tslib": "^1.7.1",
     "walk-sync": "^0.3.2",
-    "yargs": "^6.6.0",
-    "yarn": "^0.24.5"
+    "yargs": "^8.0.2",
```

Comment:
I thought there were some non-trivial changes between yargs 6 -> 8. Have you verified that everything still works here @seawatts?